tools/challenge-editor/client/src/components/editor/editor.tsx

```diff
@@ -87,6 +88,14 @@ const Editor = () => {
       <p>
         <Link to={`/${superblock || ''}/${block || ''}`}>Return to Block</Link>
       </p>
+      <p>
+        <Link
+          to={`https://freecodecamp.org/learn/${superBlockNameMap[superblock || '']}/${block || ''}/${items.name.replace(/[\s]+/g, '-').toLowerCase() || ''}`}
```

Comment:
Wouldn't it be more beneficial to link to the localhost URL so that you could quickly see your changes being applied?

Reply:
You make a good point.